README.md

```diff
@@ -2,7 +2,8 @@
 
 | Windows Build Status |
 | -------------------- |
-| [![Build status](https://ci.appveyor.com/api/projects/status/9hb88yawy54b0086?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu-manager) |
+| [![Build status](https://ci.appveyor.com/api/projects/status/9hb88yawy54b0086/branch/master?svg=true)](https://ci.appveyor.com/project/xqemu-bot/xqemu-manager?branch=master) |
+| [Latest Build](https://ci.appveyor.com/api/projects/xqemu-bot/xqemu-manager/artifacts/xqemu-manager.zip?branch=master) |
```

Comment:
Ideally this would have said something about "**Download for Windows**"; I'm not sure about this second row either.

Or rather, this link should be on our website instead. GitHub is not to be used by Users, so there's no point in distributing end-user binaries here. Also see https://github.com/xqemu/xqemu.com/issues/18

Anyhow, the important thing is that the links point to the right thing.

Thanks!